source/GameInterface/Services/MobileParties/Interfaces/MobilePartyInterface.cs

```diff
@@ -106,7 +106,7 @@ public void StartPlayerSettlementEncounter(string partyId, string settlementId)
             return;
         }
 
-        if (PlayerEncounter.Current is not null) return;
+       // if (PlayerEncounter.Current is not null) return;
```

Comment:
does this need to be commented out?

Reply:
I tested it both ways and it also seems to be related to the bug.